src/mock-data/kyc.ts

```diff
@@ -40,6 +43,21 @@ const personalDataAndDocumentsAustria: AddKycParams<KycSchema.PersonalDataAndDoc
     },
   }
 
+const {
+  identificationDocument: identificationDocumentAustria,
+  ...remainingInformationAustria
+} = personalDataAndDocumentsAustria.data
+const personalDataAndDocumentsDetailedAustria: AddKycParams<KycSchema.PersonalDataAndDocumentsDetailed> =
+  {
+    kycSchemaName: KycSchema.PersonalDataAndDocumentsDetailed,
+    data: {
+      ...remainingInformationAustria,
+      email: 'someemail@gmail.com',
+      identificationDocumentType: IdentificationDocumentType.PAS,
+      identificationDocumentFront: 'Place base64 string of document here',
```

Comment:
Yes, that was what we originally thought, since we also had to change, for example, the fiat-account mock, due to the iban there being just a string with value `mock-iban-number`, which didn't work for us because we also have IBAN validation.
Nevertheless, on this mock I can just put a base64 string of a plant or something as you suggested, since the KYC provider validates it is an image but doesn't validate the content in the sandbox. Thanks for the tip!

Reply:
thanks. if you need a better mock IBAN feel free to submit a PR for that-- probly goes without saying but make sure not to post anything sensitive since this is open source.